### PR TITLE
Week 5 Problem Set

### PR DESCRIPTION
**Pull request by jolsenvaldez**
Here is my Week 5 Problem Set: 

https://mybinder.org/v2/gh/CUB-Computational-Tools/2020-ps-jolsenvaldez/binder-python?urlpath=git-pull?repo=https%3A%2F%2Fgithub.com%2FCUB-Computational-Tools%2F2020-ps-jolsenvaldez%26branch%3Dweek5%26urlpath%3Dlab

### CODE REVIEW

File: week5.ipynb [hunk 1]
```diff
@@ -0,0 +1,366 @@
+{
```
Comment (vaga3461) [inline week5.ipynb:1]: <p>This intro section looks good! Only thing is that I notice some words capitalized that I don't think need to be; if the intention is to provide emphasis on those terms as vocab words, maybe use italics or boldface instead?</p><p> </p><p>Also, in the paragraph that starts "In this form, the S-B equation compares...", you're missing a closing parenthesis in the second sentence.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-jolsenvaldez/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (vaga3461) [inline week5.ipynb:1]: <p>I think you can shorten this line of code a bit by doing <code>constants['Radius1'] ** 2</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-jolsenvaldez/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (vaga3461) [inline week5.ipynb:1]: <p>Looks like this section got copied and pasted twice</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-jolsenvaldez/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (vaga3461) [inline week5.ipynb:1]: <p>I like the lines separating questions! How'd you do that?</p><p> </p><p>On question 2, you say "Create a variable and list of values of your own..." - I think what you want them to create is a dictionary (which is different from a list in python).</p><p> </p><p>In question 4, the conversion from Kelvin to C could be formatted using LaTeX</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-jolsenvaldez/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (vaga3461) [inline week5.ipynb:1]: <p>same comment as above for question 2 - what they create is a dictionary, not a list</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-jolsenvaldez/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (vaga3461) [inline week5.ipynb:1]: <p>In question 4, between steps 2 and 3, you cancel piR^2. But what happens to the 4 in the denominator (with 4piR^2?) I'm probably missing something really obvious...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-jolsenvaldez/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (vaga3461) [inline week5.ipynb:1]: <p>nice job, Juliana!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-jolsenvaldez/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (jolsenvaldez) [reply week5.ipynb:1]: Are you referring to "Austrian" or do you mean in the Background section when I have "Watts", etc.?

**Pull request merged**